packages/browser-crawler/src/internals/browser-crawler.ts

```diff
@@ -593,9 +594,10 @@ interface EnqueueLinksInternalOptions {
     page: CommonPage;
     requestQueue?: RequestQueue;
     defaultBaseUrl?: string;
+    transformRequestFunction?: RequestTransform;
 }
 
-export async function browserCrawlerEnqueueLinks({ options, page, requestQueue, defaultBaseUrl }: EnqueueLinksInternalOptions) {
+export async function browserCrawlerEnqueueLinks({ options, page, requestQueue, defaultBaseUrl, transformRequestFunction }: EnqueueLinksInternalOptions) {
```

Comment:
Any reason for this? You can already pass it down in options, right?

Reply:
you're right, I was kinda trying to quickly fix the test and missed it. Will update (but it would probably change with other changes, not sure yet). Thanks for noticing!